libs/sql-parser/src/main/antlr/SqlBase.g4

```diff
@@ -134,8 +134,15 @@ query
 queryNoWith
     : queryTerm
       (ORDER BY sortItem (',' sortItem)*)?
-      (LIMIT (limit=parameterOrInteger | ALL))?
-      (OFFSET offset=parameterOrInteger (ROW | ROWS)?)?
+      (limitClause? offsetClause? | offsetClause? limitClause?)?
```

Comment:
I think this could result in consecutive offsets or limits, `select * from t limit 10 limit 10`

Reply:
I can add this test, but it's not:
```
@Test
public void testParseErrorMultipleLimits() {
    assertThrowsMatches(
        () -> SqlParser.createStatement("select fuu from dual limit 1 limit 2 limit 3"),
        ParsingException.class,
        "line 1:30: mismatched input 'limit' expecting {<EOF>, ';'}");
    assertThrowsMatches(
        () -> SqlParser.createStatement("select fuu from dual fetch first 1 rows only fetch first 2 rows only"),
        ParsingException.class,
        "line 1:46: mismatched input 'fetch' expecting {<EOF>, ';'}");
}
```